packages/theming/apple-theme/src/appleTheme.macos.ts

```diff
@@ -46,6 +46,28 @@ export const appleComponents = {
       variant: 'bodyStandard',
     },
   },
+  Checkbox: {
+    checkbox: {
+      style: {
+        borderStyle: 'solid',
+        borderWidth: 0.5,
+        borderRadius: 3,
+        minHeight: 14,
+        minWidth: 14,
+        marginEnd: 5,
+      },
+    },
+    checkmarkIcon: {
+      width: 10,
+      height: 10,
+      style: {
+        marginVertical: 2,
+        marginHorizontal: 2,
+      },
+    },
+    // This disables other available states like: hovered, focused, pressed.
+    _precedence: ['disabled', 'boxAtEnd', 'checked'],
```

Comment:
Good catch! What's "boxAtEnd?"

Reply:
`boxAtEnd` adds `marginStart:0` instead of `marginEnd:0` for the checkbox square

Reply:
Interesting.. and I guess that wasn't something we couldn't have handled with just the style prop and needed this _precendence thing?

Reply:
Yep, could be handled with style prop. (@JasonVMo's version with new framework addressed this).

Reply:
OK. I guess I'll worry about that for a future review with the new compose checkbox then!